docs/reference/docs/reindex.asciidoc

```diff
@@ -478,6 +478,10 @@ which defaults to a maximum size of 100 MB.
 `sort`:::
 (Optional, list) A comma-separated list of `<field>:<direction>` pairs to sort by before indexing.
 Use in conjunction with `max_docs` to control what documents are reindexed.
+deprecated[7.6, sort in reindex is deprecated] *NOTE: Sorting in reindex is deprecated.*
+Sorting in reindex was never guaranteed to index
+documents in order and prevents future evolution of reindex such as resilience and performance
```

Comment:
I'd go with "further" rather than "future"
```suggestion
documents in order and prevents further development of reindex, such as resilience and performance
```